conans/test/utils/tools.py

```diff
@@ -559,7 +559,7 @@ def _handle_cli_result(self, command, assert_error, error):
                 output_header='{:-^80}'.format(" Output: "),
                 output_footer='-' * 80,
                 cmd=command,
-                output=str(self.stderr) + str(self.stdout)
+                output=str(self.stderr) + str(self.stdout) + "\n" + str(self.out)
```

Comment:
Why is this necessary? It shouldn't, as ``self.out = self.stderr + self.stdout``?

Reply:
I think the `run_command` (not the `run`) is not showing correctly the output error because it is doing: `with redirect_output(self.out):`.
Probably it has to be fixed otherwise.